libbeat/publisher/publish.go

```diff
@@ -176,6 +176,12 @@ func (publisher *PublisherType) Init(
 	publisher.disabled = *publishDisabled
 	if publisher.disabled {
 		logp.Info("Dry run mode. All output types except the file based one are disabled.")
+	} else if len(configs) == 0 {
+		logp.Info("No output set. Fall back to standard output elasticsearch on localhost:9200")
```

Comment:
What is the impetus for this change? I think I like the existing behavior because it forces the user to fix their configuration file. Changing libbeat to default to `localhost:9200` could mask typos in the configuration file. Consider the following snippit:

```
outputs:
  elasticsearch:
    hosts: ["192.168.99.100:9200"]
```

That would result in `localhost:9200` because of a typo (`outputs` instead of `output`).